bootstrap.js

```diff
@@ -22,6 +22,16 @@ function registerAddonResourceHandler(data) {
   return "resource://" + resourceName + "/";
 }
 
+function getBaseLoader() {
+  try {
+    // Firefox 57 and newer, use the base-loader from DevTools.
+    return Cu.import("resource://devtools/shared/base-loader.js", {});
+  } catch (e) {
+    // Firefox 56 and older, use the addon-sdk loader.
+    return Cu.import('resource://gre/modules/commonjs/toolkit/loader.js').Loader;
```

Comment:
Nit: Use " (The old version was wrong too...)

Reply:
Good catch, thanks!